pkg/tenant/db/errors.go

```diff
@@ -34,7 +34,8 @@ var (
 	ErrInvalidTopicName   = errors.New("invalid topic name")
 
 	// Database state errors
-	ErrMemberExists = errors.New("member already exists")
+	ErrMemberExists       = errors.New("member already exists")
+	ErrInvalidMemberEmail = errors.New("member does not exist")
```

Comment:
Should this just be `ErrMemberEmailNotFound` to prevent confusing it with email validation?